roles/docker/molecule/resources/verify.yml

```diff
@@ -18,7 +18,7 @@
         volumes: true
         images: true
         containers: true
-        disk_usage: true
+        disk_usage: false
```

Comment:
The [Molecule test for the `docker` role fails](https://github.com/UCL-MIRSG/ansible-collection-infra/actions/runs/19827063023/job/56803077975#step:3:4231) when attempting to check the disk usage on RockyLinux 9.